segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -162,22 +124,21 @@ namespace pcl
         exclude_labels_ = boost::make_shared<std::vector<bool> >(exclude_labels);
```

Comment:
Agreed. Didn't change originally because it's not breaking anything at this point and it would be an additional thing breaking the API.